core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -300,7 +300,6 @@ class KiwixTextToSpeech internal constructor(
               bundle,
               bundle.getString(Engine.KEY_PARAM_UTTERANCE_ID)
             )
-            currentPiece.getAndIncrement()
```

Comment:
@MohitMaliDeveloper I don't think we should remove that.

Reply:
@gouri-panda can you please share your analysis on this? why you are not in favor of removing this?

As I see in the code we are already incrementing the index 

https://github.com/kiwix/kiwix-android/blob/bd31240002ed417ed9e9587ae088a17198eefab2/core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt#L298

so I don't understand why we are again incrementing the index it will skip the next line. This is happening when I try to read the article, if I keep this then it is skipping the next line.

Reply:
Ohh! I see now @MohitMaliFtechiz That was a bug. We didn't notice earlier.